packages/tenant-process/test/utils.ts

```diff
@@ -195,6 +197,10 @@ export const addOneEService = async (eservice: EService): Promise<void> => {
   await writeInReadmodel(toReadModelEService(eservice), eservices);
 };
 
+export const addOneAttribute = async (agreement: Attribute): Promise<void> => {
+  await writeInReadmodel(toReadModelAttribute(agreement), attributes);
+};
```

Comment:
Typo, this is not an `agreement`. By the way, we could avoid this function entirely: it just performs a call to another function.

Reply:
Ups, fixes, thanks

Reply:
I prefer using this function since abstracts away the need of importing the attributes collection object, and make the `toReadModel` convertion